example-dvc-experiments/generate.bash

```diff
@@ -87,61 +64,70 @@ git tag "git-init"
 
 cp -r "${HERE}"/code/src .
 cp "${HERE}"/code/requirements.txt .
+cp "${HERE}"/code/requirements-macos.txt .
 cp "${HERE}"/code/params.yaml .
+## We are assuming the repo is generated in Linux
+## Otherwise the following line must be changed to have requirements-macos.txt
```

Comment:
can we detect this?

Reply:
Yes, we can detect it. Would you like to support Windows as well?

Reply:
Added support with `uname -s` for macOS only. If you'd like Windows support, let me know.